data: RWUBAKWANAYO

RWUBAKWANAYO opened the pull request:
<!-- If your PR fixes an open issue, use `Closes #999` to link your PR with the issue. #999 stands for the issue number you are fixing -->

## Fixes Issue

<!-- Remove this section if not applicable -->

<!-- Example: Closes #31 -->

## Changes proposed

<!-- List all the proposed changes in your PR -->

<!-- Mark all the applicable boxes. To mark the box as done follow the following conventions -->
<!--
[x] - Correct; marked as done
[X] - Correct; marked as done

[ ] - Not correct; marked as **not** done
-->

## Check List (Check all the applicable boxes) <!-- Follow the above conventions to check the box -->

- [ ] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [ ] This PR does not contain plagiarized content.
- [ ] The title of my pull request is a short description of the requested changes.

## Screenshots

![validate json](https://user-images.githubusercontent.com/68381641/231705362-e94cd15e-54a2-4c3d-b618-dd066bbd61ec.png)

## Note to reviewers

In this pull request, I followed the official [documentation](https://linkfree.eddiehub.io/docs/quickstart) to create my profile for the first time, including my name, a summary of myself, and links to my social media accounts.


<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/6047"><img src="https://gitpod.io/button/open-in-gitpod.svg"/></a>

khushalarora reviewed (changes requested):
Hi @RWUBAKWANAYO, thank you for contributing :tada: 
Please rename your file and add **`data/`** in the beginning :+1: 
Your filename will look like  **`data/RWUBAKWANAYO.json`**

RWUBAKWANAYO commented:
> Hi @RWUBAKWANAYO, thank you for contributing 🎉 Please rename your file and add **`data/`** in the beginning 👍 Your filename will look like **`data/RWUBAKWANAYO.json`**

Thanks for the suggestions. I've updated the file name and added data/ in the beginning.

SaraJaoude reviewed (approved):
looks good
